amulet/helpers.py

```diff
@@ -67,7 +67,7 @@ def juju(args, env=None, include_model=True):
         else:
             args = list(args)
             # insert the model arg after the command, but before any other args
```

Comment:
Someone once said, but I don't recall who:  "Comments are lies.  If they're not today, they will be." :-)